client/my-sites/plans-features-main/index.jsx

```diff
@@ -92,6 +96,17 @@ const jetpackProducts = [
 	},
 ];
 
+const jetpackProductPriceMatrix = {
```

Comment:
I think this definition could be moved to [`products-values/constants.js`](https://github.com/Automattic/wp-calypso/blob/d6c5644ad5381a537f143816d7b4d1744b927756/client/lib/products-values/constants.js) as I don't see it being dependant on `PlansFeaturesMain` in any way.

It's not a blocker in merging, though, so leaving it as a suggestion.

Reply:
Good call, thanks! Addressed in #37104